File: 2350-find-closest-number-to-zero/find-closest-number-to-zero.py
class Solution:
    def findClosestNumber(self, nums: List[int]) -> int:
        closest = nums[0]
        for num in nums:
            if abs(num) == abs(closest):
                if num > closest:
                    closest = num
            if abs(num) < abs(closest):
                closest = num
        return closest

File: 2350-find-closest-number-to-zero/find-closest-number-to-zero.java
class Solution {
    public int findClosestNumber(int[] nums) {
        int ans = nums[0];
        for (int num : nums) {
            if (Math.abs(num) <= Math.abs(ans)) {
                if (Math.abs(num) == Math.abs(ans))
                    ans = (num > ans) ? num : ans;
                else
                    ans = num;
            }
        }
        return ans;
    }
}